gandalf.js

```diff
@@ -1,21 +1,20 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?